### PR TITLE
Update form validation, side nav and tabs stories to CSF format

### PR DESCRIPTION
**Pull request by elenagarrone**
# Description

Update form validation, side nav and tabs stories to CSF format

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts

### CODE REVIEW

File: projects/canopy/src/lib/forms/validation/form.notes.ts [hunk 1]
```diff
@@ -18,7 +15,7 @@ Import the module in your core component, e.g.:
 ~~~js
 @NgModule({
   ...
-  imports: [LgValidationModule]
+  imports: [..., LgValidationModule]
```
Comment (paul-request) [inline projects/canopy/src/lib/forms/validation/form.notes.ts:18]: Is this a mistake or some new voodoo I'm not aware of?

File: projects/canopy/src/lib/tabs/tab-nav-bar.stories.ts [hunk 2]
```diff
@@ -1,42 +1,134 @@
...
-    ],
-    notes: {
-      markdown: tabbedNavNotes,
+    docs: {
+      description: {
+        component: tabbedNavNotes,
+      },
+    },
+  },
+  argTypes: {
```
Comment (paul-request) [inline projects/canopy/src/lib/tabs/tab-nav-bar.stories.ts:23]: 🤮 would love a better way of doing this

Comment (elenagarrone) [reply projects/canopy/src/lib/tabs/tab-nav-bar.stories.ts:23]: It's so bad 🙄

File: projects/canopy/src/lib/forms/validation/form.notes.ts (hunk 1 above)
Comment (elenagarrone) [reply projects/canopy/src/lib/forms/validation/form.notes.ts:18]: It was done on purpose as most of the other stories have it.. it's just to say that a user might have more imports - it's just the notes file ;)

**Pull request merged**